files/scripts/syncd.sh

```diff
@@ -104,11 +104,12 @@ start() {
         if [[ x"$WARM_BOOT" != x"true" ]]; then
             if [[ x"$(/bin/systemctl is-active pmon)" == x"active" ]]; then
                 /bin/systemctl stop pmon
-                /usr/bin/hw-management.sh chipdown
-                /bin/systemctl restart pmon
-            else
-                /usr/bin/hw-management.sh chipdown
+                debug "pmon is active while syncd starting, stop it first"
             fi
+            /usr/bin/hw-management.sh chipdown
+            debug "Starting pmon service..."
+            /bin/systemctl restart pmon
+            debug "Started pmon service"
```

Comment:
"pmon service started"

Reply:
> "pmon service started"

In order to be consistent with other services.

Reply:
@stephenxs looks like these changes prevent pmon restart in case of config reload after warm-boot. Not sure if this is desired behaviour.